apps/storybook-e2e/src/e2e/elements/input-field/vl-input-field.stories.cy.ts

```diff
@@ -3,6 +3,7 @@ const inputFieldUrl = 'http://localhost:8080/iframe.html?id=elements-input-field
 describe('story vl-input-field - default', () => {
     it('should be accessible', () => {
         cy.visitWithA11y(`${inputFieldUrl}`);
+
         cy.get('[is="vl-input-field"]').should('have.class', 'vl-input-field');
```

Comment:
Kan je hier ook de cy,checkA11y toevoegen?

Reply:
toegevoegd;
ik heb nu wel de story aangepast zodat de accessibility check zou valideren (een label toegevoegd) maar ik denk dat dit ok is